examples/visual-tests/amp-inabox/amp-inabox-gpt.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>AMP Inabox GPT Test</title>
+<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
+<script>
+var googletag = googletag || {};
+googletag.cmd = googletag.cmd || [];
+</script>
+
+<script>
+googletag.cmd.push(function() {
+  googletag.defineSlot('/8479/regtests/AmpInaboxTest.java/firstad', [300, 250], 'div-gpt-ad-1541535279148-0').addService(googletag.pubads());
+  googletag.pubads().enableSingleRequest();
+  googletag.enableServices();
+  googletag.pubads().addEventListener('slotRenderEnded', function(event) {
+    document.getElementById('slotrenderendedannounce').textContent = "SlotRenderEnded fired.";
+    document.documentElement.classList.add('slot-render-ended');
+  });
+  /*
```

Comment:
Don't leave dead code in the main repo... that's what private Git branches are for :)